builtin/logical/pki/path_roles.go

```diff
@@ -286,6 +286,11 @@ for "generate_lease".`,
 				Type:        framework.TypeBool,
 				Description: `Mark Basic Constraints valid when issuing non-CA certificates.`,
 			},
+			"not_before": &framework.FieldSchema{
+				Type:        framework.TypeInt,
```

Comment:
Can you switch this to TypeDuration which will require some other changes in storage.  See max_ttl for an example of a TypeDuration field.